rskj-core/src/main/java/co/rsk/pcc/altBN128/impls/JavaAltBN128.java

```diff
@@ -124,8 +120,7 @@ public int pairing(byte[] data, int length) {
 
             // fail if decoding has failed
             if (pair == null) {
-                output = EMPTY_BYTE_ARRAY;
-                return 1;
+                return returnError();
```

Comment:
Maybe out of scope, why not make decodePair static?

Reply:
👍